tests/plugins/uploadwidget/manual/fileformatnotification.html

```diff
@@ -0,0 +1,22 @@
+<div id="editor"></div>
+
+<script>
+	CKEDITOR.plugins.add( 'uploadplugin', {
+		requires: 'uploadwidget',
+		init: function( editor ) {
+			CKEDITOR.fileTools.addUploadWidget( editor, 'uploadplugin', {
+				supportedTypes: /image\/(jp2)/,
+			} );
+		}
+	} );
+
+	CKEDITOR.replace( 'editor', {
+		language: 'en',
+		extraPlugins: 'uploadplugin',
+		on: {
+			instanceReady: function( evt ) {
+				bender.tools.ignoreUnsupportedEnvironment( 'uploadimage' );
```

Comment:
Shouldn't it use the method for the `uploadwidget` plugin?

```suggestion
				bender.tools.ignoreUnsupportedEnvironment( 'uploadwidget' );
```

The same issue is present in other manual tests.

Reply:
Yes, it should match the specific plugin namespace, in reality, these are using the same condition for unsupported environment check but it's misleading to use a different plugin name than the system under test.